### PR TITLE
Execute units tests against php 8.1 & 8.2

### PR DESCRIPTION
**Pull request by Fenn-CS**
Executes unit tests against more recent php versions on master and stable27

Resolves : https://github.com/nextcloud/logreader/issues/868

### CODE REVIEW

File: .github/workflows/phpunit-mysql.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
     strategy:
       matrix:
         php-versions: ['8.0', '8.1', '8.2']
-        server-versions: ['master']
+        server-versions: ['master', 'stable27']
```
Comment (susnux) [inline .github/workflows/phpunit-mysql.yml:42]: Why `stable27` this is only for the master branch, `stable27` should only be tested on its own branch.
(so backporting this to `stable27` instead)

**Review by susnux — CHANGES_REQUESTED** [review]

File: .github/workflows/phpunit-mysql.yml (hunk 1 above)
Comment (Fenn-CS) [reply .github/workflows/phpunit-mysql.yml:42]: @susnux the original issue : https://github.com/nextcloud/logreader/issues/868 requested such?

Comment (susnux) [reply .github/workflows/phpunit-mysql.yml:42]: I think that is about adding 8.2 also to the `stable27` branch.

Comment (nickvergessen) [reply .github/workflows/phpunit-mysql.yml:42]: yes, what susnux says

File: .github/workflows/phpunit-oci.yml [hunk 2]
```diff
@@ -38,8 +38,8 @@ jobs:
 
     strategy:
       matrix:
-        php-versions: ['8.0']
-        server-versions: ['master']
+        php-versions: ['8.0', '8.1', '8.2']
```
Comment (nickvergessen) [inline .github/workflows/phpunit-oci.yml:41]: this should also not be changed, to keep the CI matrix small.

**Review by nickvergessen — CHANGES_REQUESTED** [review]
Basically https://github.com/nextcloud/logreader/pull/877 fixed master,
but the change should be either backported to stable27, or you manually add 8.2 to all phpunit-* files tehre

**Comment by Fenn-CS** [comment]
Closing this with : https://github.com/nextcloud/logreader/pull/916

**Pull request closed without merging**